automation/recipes/meteor-qpsk.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# See: https://www.reddit.com/r/RTLSDR/comments/abn29d/automatic_meteor_m2_reception_on_linux_using_rtl/ed2teuv/
+# Meteor-M 2 satellite reception
+
+BASENAME=$1
```

Comment:
There should be a header comment. Explaining what the script is for and what are the parameters expected (and their format, e.g. whether timeout is in seconds, ms, unit time or maybe ISO time).